core/src/main/java/hudson/cli/GroovyCommand.java

```diff
@@ -59,7 +59,7 @@ public String getShortDescription() {
 
     protected int run() throws Exception {
         // this allows the caller to manipulate the JVM state, so require the execute script privilege.
-        Jenkins.get().checkPermission(Jenkins.RUN_SCRIPTS);
+        Jenkins.getActiveInstance().checkPermission(Jenkins.ADMINISTER);
```

Comment:
why is this change to active instance required?

Reply:
I believe this branch originated before the removal of the deprecated method was in master, fixed now